package.json

```
@@ -17,7 +17,7 @@
     "serve": "NODE_PATH=./dist node dist/server.js",
     "watch-node": "NODE_PATH=./dist nodemon dist/server.js",
     "watch": "concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"cyan.bold,green.bold\" \"yarn watch-ts\" \"yarn watch-node\"",
-    "test": "DISABLE_LOGS=true jest --forceExit --coverage --verbose",
+    "test": "DISABLE_LOGS=true jest item.test.ts --forceExit --coverage --verbose",
```

We probably want to revert this
```suggestion
    "test": "DISABLE_LOGS=true jest --forceExit --coverage --verbose",
```